project-resources/requesting_LFIT_support.md

```diff
@@ -1,45 +1,43 @@
-# Requesting Linux Foundation IT Support 
+# Requesting Linux Foundation IT Support Services 
 
-This guide outlines the procedures and best practices for requesting assistance from the Linux Foundation IT Operations team.
+## Introduction
+This guide is intended for OpenJS projects that have formally engaged with the Linux Foundation IT team to support one or more of their services. It outlines the available IT support services, how to submit a request and the process for escalating issues.
 
-## Who Is This Document For?
+## Services Available
+ - Cloud Infrastructure Services, Consulting, and Operational Support
+ - CDN Implementation, Optimization, and Support Services
+ - Infrastructure and Application Monitoring Services
+ - Custom Continuous Integration and Delivery Consultation and Services
+ - CI Workflow Design, Optimization, and Consulting Services
+ - Artifact Repository Management and Consulting Services
+ - Repository Management and Consultation Services
+ - Domain Management Services
+ - DNS Management and Consulting Services
+ - SSL/TLS Certificate Management Services
+ - Code Signing Certificate Management Services
+ - Jira Configuration, Customization, and Support Services
+ - Confluence Configuration, Customization, and Support Services
+ - Mailing List Management and Support Services
+ - Meeting Management and Support Services
 
-This document is for OpenJS projects that have formally engaged with LF IT to support one or more of their services. If you have questions about a non-LF supported service, please contact operations@openjsf.org.
+## How to Submit a Request
 
-## How To Contact Linux Foundation IT For Support
+The Linux Foundation provides support through Jira Service. Submitting requests via Jira ensures that they are properly tracked and assigned.
```

Comment:
The Linux Foundation provides support through our IT Support Desk. Submitting requests via the support desk ensures that they are properly tracked and assigned.

Reply:
Ideally you should just link to https://docs.releng.linuxfoundation.org/en/latest/helpdesk.html as that's the documentation for all of our projects. IMO that is better than spelling out the process in this document and in another project's document, etc...